editor/js/Loader.js

```diff
@@ -438,12 +438,27 @@ function Loader( editor ) {
 					var { PLYLoader } = await import( '../../examples/jsm/loaders/PLYLoader.js' );
 
 					var geometry = new PLYLoader().parse( contents );
-					var material = new THREE.MeshStandardMaterial();
+					var object;
 
-					var mesh = new THREE.Mesh( geometry, material );
-					mesh.name = filename;
+					if ( geometry.index !== null ) {
 
-					editor.execute( new AddObjectCommand( editor, mesh ) );
+						var material = new THREE.MeshStandardMaterial();
+
+						object = new THREE.Mesh( geometry, material );
+						object.name = filename;
+
+					} else {
+
+						var material = new THREE.PointsMaterial( { size: 0.01 } );
+
+						if ( geometry.hasAttribute( 'color' ) === true ) material.vertexColors = true;
```

Comment:
maybe
```js
material.vertexColors = geometry.hasAttribute( 'color' );
```
as is done elsewhere in the file.

Reply:
Good catch!